angr/analyses/cfg_base.py

```diff
@@ -1226,7 +1226,11 @@ def _process_irrational_functions(self, functions, predetermined_function_addrs,
 
             if not has_unresolved_jumps:
                 continue
-
+            
+            if function.startpoint == None:
```

Comment:
`function.startpoint` becomes `None` in the following case:

a) CFGFast scans Function A, which is not properly terminated, and goes all the way through into Function B. If at this moment we don't know the existence of Function B at all (i.e. we never see a call to Function B, and the address of Function B is not provided to us in any form), all blocks in Function B will be considered to be part of Function A.
b) Later we see a call to Function B, but unfortunately we have scanned all blocks in Function B and generously assign them to Function A. Function B will be an empty function for now.

However Function B is only temporarily empty. It will be fixed during the post-analysis procedure where we revisit the entire graph and re-create all functions.